packages/react-test-renderer/src/__tests__/ReactTestRendererAsync-test.js

```diff
@@ -46,7 +46,7 @@ describe('ReactTestRendererAsync', () => {
 
   it('flushAll returns array of yielded values', () => {
     function Child(props) {
-      renderer.unstable_yield(props.children);
+      ReactTestRenderer.unstable_yield(props.children);
```

Comment:
I get why it's appealing for yield not to be tied to a root, but it seems odd to me since yielding and flushing are super coupled, and flushing _is_ necessarily tied to a root.